package.json

```diff
@@ -1,38 +1,43 @@
 {
-  "name": "cesium-webpack-example",
+  "name": "cesiumjs-webpack-example",
   "version": "1.0.0",
   "description": "The minimal recomended setup for an app using Cesium with Webpack.",
-  "main": "index.js",
+  "homepage": "https://cesium.com/platform/cesiumjs/",
+  "license": "Apache-2.0",
+  "author": {
+    "name": "Cesium GS, Inc.",
+    "url": "https://cesium.com"
+  },
   "keywords": [
-    "cesium",
+    "CesiumJS",
     "webpack",
     "example"
   ],
-  "author": {
-    "name": "Cesium GS, Inc.",
-    "url": "https://cesium.com"
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/CesiumGS/cesium-webpack-example.git"
   },
-  "homepage": "https://cesiumjs.org",
-  "license": "Apache-2.0",
-  "dependencies": {
-    "cesium": "^1.63.1"
+  "main": "index.js",
+  "scripts": {
+    "build": "node_modules/.bin/webpack --config webpack.config.js",
+    "start": "node_modules/.bin/webpack serve --config webpack.config.js --open"
   },
   "devDependencies": {
-    "copy-webpack-plugin": "^6.0.3",
-    "css-loader": "^3.2.0",
-    "html-webpack-plugin": "^3.2.0",
+    "cesium": "^1.84.0",
+    "copy-webpack-plugin": "^9.0.1",
+    "css-loader": "^6.2.0",
+    "html-webpack-plugin": "^5.3.2",
+    "sass-loader": "^7.1.0",
     "strip-pragma-loader": "^1.0.0",
-    "style-loader": "^1.0.0",
-    "uglifyjs-webpack-plugin": "^2.2.0",
-    "url-loader": "^2.2.0",
-    "webpack": "^4.41.2",
-    "webpack-cli": "^3.3.9",
-    "webpack-dev-server": "^3.9.0"
+    "style-loader": "^3.2.1",
+    "url-loader": "^4.1.1",
+    "webpack-cli": "^4.8.0",
+    "webpack-dev-server": "^3.11.2"
   },
-  "scripts": {
-    "build": "node_modules/.bin/webpack --config webpack.config.js",
-    "start": "node_modules/.bin/webpack-dev-server --config webpack.config.js",
-    "release": "node_modules/.bin/webpack --config webpack.release.config.js",
-    "serve-release": "node_modules/.bin/webpack-dev-server --config webpack.release.config.js --content-base /dist"
+  "dependencies": {
+    "cesiumjs": "^1.10.0",
```

Comment:
Thanks for catching this. I was a little bit confused because we are using two different names to import CesiumJS - "cesium" and "cesiumjs"
Regardless, I removed the `"cesiumjs": "^1.10.0"` dependency.